nbgrader/tests/apps/test_nbgrader_autograde.py

```diff
@@ -9,6 +9,17 @@
 from .. import run_nbgrader
 from .base import BaseTestApp
 
+PLUGIN = """
```

Comment:
Since this is only used in one place, I would define it in the function where you use it rather than having it be a global variable.